src/modules/device.json

```diff
@@ -255,7 +263,8 @@
           },
           "required": [
             "sdk",
-            "os"
```

Comment:
It seems like removing a property that a client might be using is a breaking change, no?

Reply:
i guess you’re right.

Reply:
I think there is some precedent for releasing breaking changes prior to a `1.0` release, but that's an expectation I would want to set with consumers prior to surprising them.